src/Package/MSTest.Sdk/Sdk/Sdk.props

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Project>
+  <!-- Implicit top import -->
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  
+  <PropertyGroup>
+    <UseMSTestRunner>true</UseMSTestRunner>
+    <TestingPlatformDonetTestSupport>false</TestingPlatformDonetTestSupport>
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+    <LangVersion>preview</LangVersion>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
this one maybe should be in the project self like asp.net template
```xml
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>net8.0</TargetFramework>
    <Nullable>enable</Nullable>
    <ImplicitUsings>enable</ImplicitUsings>
  </PropertyGroup>

</Project>
```

Reply:
Yep I would prefer to have these coming from user project (`LangVersion` + `ImplicitUsings`)